tests/test_license.py

```diff
@@ -167,7 +168,7 @@ def getvalue(_):
                 return 'GPL-3.0'.encode('utf-8')
 
         # set the mocks
-        license.BytesIO = MockBytesIO
+        download.BytesIO = MockBytesIO
```

Comment:
Probably need to reset this I would guess as I see further down license.BytesIO is reset instead.

Reply:
Good catch. I fixed the line that resets it below.